Sources/XCTestDynamicOverlay/XCTUnimplemented.swift

```diff
@@ -0,0 +1,411 @@
+// MARK: (Parameters) -> Void
+
+/// Returns a closure that generates a failure when invoked.
+///
+/// - Parameter description: An optional description of the unimplemented closure, for inclusion in
+///   test results.
+/// - Returns: A closure that generates a failure when invoked.
+@_disfavoredOverload
+public func XCTUnimplemented(
```

Comment:
There are a _lot_ of overloads in this file, but I'm not sure it can be helped because the returned closure needs to be rather concrete.